CHANGELOG

```diff
@@ -1,6 +1,13 @@
 devel
 -----
 
+* Fixed ES-1508: (EE only) when deleting edges in a SmartGraph via
+  DELETE /_api/document/{collection} using _key or _id values as
+  document selectors, the INBOUND and OUTBOUND caches of the
```

Comment:
I would like to rephrase this: if I understand correctly, the issue does not affect the "caches" (in the sense that we use the term cache elsewhere, i.e. in-memory cache), but rather the `_from` and `_to` collections of the SmartGraph. If that understanding is correct, let's please adjust the CHANGELOG so that readers don't draw any false conclusions from it.
```suggestion
  document selectors, the INBOUND and OUTBOUND caches of the
```